projects/ngx-d3timeline/src/lib/hover/hover-event.ts

```diff
@@ -0,0 +1,29 @@
+import { identifier } from '../core/types';
+import {
+  Activity,
+  getActivityFromPositionedActivity
+} from '../activity/activity';
+import { PositionedActivity } from '../activity/positioned-activity';
+
+export enum HoverType {
```

Comment:
maybe `hoverStatus` ? hovered is not really a type of hover

Reply:
How about `HoverAction`?

Reply:
yeah thought of that, maybe a bit close to state actions, but I'm fine with it and its way better than HoverType